File: 2017-11-27/W5D1/W5D1/Person.m
//
//  Person.m
//  W5D1
//
//  Created by Jason Liang on 11/27/17.
//  Copyright © 2017 Jason Liang. All rights reserved.
//

#import "Person.h"

@implementation Person

- (instancetype)initWithName:(NSString *)name address:(NSString *)address {
  self = [super init];
  if (self) {
    self.name = name;
    self.address = address;
  }
  return self;
}

@end

File: 2018-06-28/W4D4/W4D4/Person.swift
//
//  Person.swift
//  W4D4
//
//  Created by Jason Liang on 6/28/18.
//  Copyright © 2018 jasonliang. All rights reserved.
//

import Foundation
import RealmSwift

class Person: Object {
  @objc dynamic var name = ""
  @objc dynamic var picture: Data? = nil // optionals supported
  let dogs = List<Dog>()
}
